packages/flutter/test/widgets/draggable_test.dart

```diff
@@ -2683,6 +2684,44 @@ void main() {
     await _testChildAnchorFeedbackPosition(tester: tester, left: 100.0, top: 100.0);
   });
 
+  testWidgets('Drag feedback change cursor correctly', (WidgetTester tester) async {
```

Comment:
Just a small one, but I would say that there is an `s` missing. 
`changes` should be the correct spelling 🤔

Reply:
Thanks. I will be more careful next time. 😓